src/big/big_decimal.cr

```diff
@@ -8,6 +8,17 @@ require "big"
 #
 # The general idea and some of the arithmetic algorithms were adapted from
 # the MIT/APACHE -licensed https://github.com/akubera/bigdecimal-rs
+#
+# ### Usage:
+# ```
+# require "big"
+# # From a string representation
+# BigDecimal.new("123456789123456789.123456789123456789") # => 123456789123456789.123456789123456789
```

Comment:
I would actually suggest to use smaller, comprehensible numbers in these examples. The `Big*` types are not meant only for handling big numbers but for providing the *ability* to have arbitrary large and precise numbers. Smaller numbers work perfectly fine in these examples, too.